guides/common/assembly_provisioning-cloud-instances-openstack.adoc

```diff
@@ -15,12 +15,13 @@ In {ProjectNameX}, you can interact with {OpenStack} REST API to create cloud in
 
 include::modules/snip_common-compute-resource-prereqs.adoc[]
 * A {SmartProxyServer} managing a network in your OpenStack environment.
-For more information, see {ProvisioningDocURL}Configuring_Networking[Configuring Networking].
+For more information, see {ProvisioningDocURL}Configuring_Networking_provisioning[] in the _Provisioning_ guide.
```

Comment:
I'm adding the "resolved" context because the anchor is defined as `guides/common/modules/con_configuring-networking.adoc:1:[id="Configuring_Networking_{context}"]`. As we link to an anchor in a different guide with a different context, we need to add `_provisioning`.